sdk/attestation/azure-security-attestation/src/test/java/com/azure/security/attestation/AttestationPolicyManagementTests.java

```diff
@@ -0,0 +1,248 @@
+     * @param clientUri URI for client - used to determine the expected response.
+     * @param claims Claims inside policy certificate token to be verified.
+     */
+    private void verifyGetPolicyCertificatesResponse(String clientUri, @NotNull JWTClaimsSet claims) {
+        assertTrue(claims.getClaims().containsKey("x-ms-policy-certificates"));
+        Object policyCertificates = claims.getClaims().get("x-ms-policy-certificates");
+        assertTrue(policyCertificates instanceof JSONObject);
+        JSONObject certificates = (JSONObject) policyCertificates;
+        assertTrue(certificates.containsKey("keys"));
+        assertTrue(certificates.get("keys") instanceof JSONArray);
+        JSONArray certificateArray = (JSONArray) certificates.get("keys");
+
+        ClientTypes clientType = classifyClient(clientUri);
+        if (clientType == ClientTypes.SHARED || clientType == ClientTypes.AAD) {
+            assertEquals(0, certificateArray.size());
+        } else {
+            assertNotEquals(0, certificateArray.size());
+            for (Object certObject : certificateArray) {
+                assertTrue(certObject instanceof JSONObject);
+                JSONObject cert = (JSONObject) certObject;
+                assertEquals("RSA", cert.get("kty").toString());
+                assertEquals("RS256", cert.get("alg").toString());
+                assertTrue(cert.containsKey("x5c"));
+                assertTrue(cert.get("x5c") instanceof JSONArray);
+                JSONArray x5c = (JSONArray) cert.get("x5c");
+                assertTrue(x5c.get(0) instanceof String);
+                /* TODO: Verify that the certificate is the isolated certificate */
+            }
+        }
+    }
+
+
+    /**
+     * Verifies attestation policy set operations.
+     *
+     * This method iterates over the attestation types and instances and attempts to set attestation policies on
+     * each instance.
+     *
+     * For AAD instances, we try set policy with two separate policies, one secured and the other unsecured.
+     * For Isolated instances, we try to set policy with a policy document signed by the isolated signer.
+     *
+     * After the policy is set, the test method cleans up by resetting attestation policy to the default policy.
+     *
+     * @param httpClient HTTP Client used for operations.
+     * @param clientUri Base URI for attestation instance.
+     * @throws JOSEException Should never be thrown.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getAttestationClients")
+    void testAddAttestationPolicyManagementCertificate(HttpClient httpClient, String clientUri)
+        throws JOSEException, InvalidKeySpecException, NoSuchAlgorithmException, ParseException, JsonProcessingException {
+
+        ClientTypes clientType = classifyClient(clientUri);
+
+        // This test only works on isolated instances.
+        if (clientType != ClientTypes.ISOLATED) {
+            return;
```

Comment:
You can use `assumeTrue("Test case only runs on ISOLATED instances.", clientType == ClientTypes.ISOLATED);` rather than this `if` clause.

Reply:
Not sure what the syntax of assumeTrue is - apparently it isn't a member of the Assertions class.

Reply:
Ah, found it - Assumptions namespace.